Cordova.podspec

```diff
@@ -0,0 +1,35 @@
+require "json"
+
+packageJson = JSON.parse(File.read(File.join(__dir__, "package.json")))
+relVersion = package['version']
+relSource  = { :git => "https://github.com/apache/cordova-ios.git",
+            :tag => "rel/{relVersion}",
+            :submodules => true }
+
+Pod::Spec.new do |s|
+  s.name         = "Cordova"
+  s.version      = relVersion
+  s.summary      = "Apache Cordova for iOS"
+  s.homepage     = "https://github.com/apache/cordova-ios"
+  s.license      = { :type => "Apache 2.0", :file => "LICENSE" }
+  s.author       = { "Apache Software Foundation" }
+  s.platform     = :ios, "10.0"
+  s.source       = relSource
+  s.requires_arc = true
+  s.default_subspec  = 'Cordova'
+  s.subspec 'Cordova' do |cordova|
+      cordova.source_files = 'CordovaLib/Classes/**/*.{h,m}', 'CordovaLib/Cordova/Cordova.h'
+      cordova.public_header_files = 'CordovaLib/Classes/Public/CDV.h', 'CordovaLib/Classes/Public/CDVAppDelegate.h', 'CordovaLib/Classes/Public/CDVAvailability.h', 'CordovaLib/Classes/Public/CDVAvailabilityDeprecated.h', 'CordovaLib/Classes/Public/CDVCommandDelegate.h', 'CordovaLib/Classes/Public/CDVCommandDelegateImpl.h', 'CordovaLib/Classes/Public/CDVCommandQueue.h', 'CordovaLib/Classes/Public/CDVConfigParser.h', 'CordovaLib/Classes/Public/CDVInvokedUrlCommand.h', 'CordovaLib/Classes/Public/CDVPlugin+Resources.h', 'CordovaLib/Classes/Public/CDVPlugin.h', 'CordovaLib/Classes/Public/CDVPluginResult.h', 'CordovaLib/Classes/Public/CDVScreenOrientationDelegate.h', 'CordovaLib/Classes/Public/CDVTimer.h', 'CordovaLib/Classes/Private/Plugins/CDVUIWebViewEngine/CDVUIWebViewDelegate.h', 'CordovaLib/Classes/Public/CDVURLProtocol.h', 'CordovaLib/Classes/Public/CDVUserAgentUtil.h', 'CordovaLib/Classes/Public/CDVViewController.h', 'CordovaLib/Classes/Public/CDVWebViewEngineProtocol.h', 'CordovaLib/Classes/Public/CDVWhitelist.h', 'CordovaLib/Cordova/Cordova.h', 'CordovaLib/Classes/Public/NSDictionary+CordovaPreferences.h', 'CordovaLib/Classes/Public/NSMutableArray+QueueAdditions.h'
+      cordova.prefix_header_contents = ''
+      cordova.requires_arc = true
```

Comment:
As mentioned in the bigger comment, I'm a bit out of touch here, but there appear to be a few items missing from this subspec that appear in the published version, namely:

 * preserve_paths - for some of the assets
 * frameworks - for some of the required Core frameworks